resources/assets/js/features/user/components/Profile/Reviews/GivenContainer.js

```diff
@@ -0,0 +1,46 @@
+import React from 'react';
+import {connect} from 'react-redux';
+import {bindActionCreators} from 'redux';
+import {setGivenReviews} from '../../../actions';
+import ReviewsList from './ReviewsList';
+import Preloader from 'app/components/Preloader'
+import {fetchGiven} from 'app/services/ReviewService'
+import ReviewsListEmpty from './ReviewsListEmpty';
+
+class GivenContainer extends React.Component {
+
+    constructor() {
+        super();
+        this.state = {
+            preloader: false
+        };
+    }
+
+    componentWillMount() {
+        this.setState({preloader: true});
+        fetchGiven().then((response) => {
+            this.setState({preloader: false});
+            this.props.setGivenReviews(response.data);
+        }).catch(() => this.setState({preloader: false}));
+    }
+
+    render() {
+        const {givenReviews} = this.props,
+            {preloader} = this.state;
+
+        return (
+            <div style={{position: 'relative'}}>
+                <Preloader enable={preloader} />
+                <ReviewsListEmpty show={givenReviews.length <= 0 && !preloader} />
```

Comment:
avoid using such statements, define variable or method that will describe this logic